WeakAuras/Prototypes.lua

```diff
@@ -6326,6 +6374,12 @@ WeakAuras.event_prototypes = {
           return state and state.show and WeakAuras.CheckRange(state.unit, needle, needle2);
         end,
       },
+      {
+        name = "rangeList",
+        type = "description",
+        display = "|cFFFFA500It is only possible to check if a unit is within the following ranges:",
+        text = function() return L["|cFFAAFFAAFriendly Units:|r|n%s|n|cFFFFAAAAHarmful Units:|r|n%s|n|cFFAAAAFFMiscellanous Units:|r|n%s|n|nThis list is based on your current character."]:format(RangeCacheStrings.friend or "", RangeCacheStrings.harm or "", RangeCacheStrings.misc or "") end
+      },
```

Comment:
There's already "L["Note: This trigger type estimates the range to the hitbox of a unit. The actual range of friendly players is usually 3 yards more than the estimate."]," above, adding this text to that is prefarable. 

Also, please less colors.

If the text contains the ranges that the lib provides, it should be prefaced with a bit more information. E.g. something like this:

The range checking capabilities depend on the current spec, class, items and type of unit that is checked. Generally the following ranges are always supported: 8, 40 [I don't actually know what this set is].

For your current setup [????], the following ranges can be checked: 
Friendly Units: 1,2,3,4,5,7,8,10, 15, 20, 25, 30, 35, 38, 40, 45, 50, ...
Harmful Units: ...
Other Units: ...
[Isn't the big distinction also players vs npcs?]

I don't think there's any value in value in showing ranges, the input us after all just one range.

Reply:
Well I didn't want to add it to the above description because it was long and seemed unsightly. I understand having a warning at the top (is that 3 yard thing even accurate?) but I don't think UI/UX-wise that the list should come before the actual functional part of the trigger personally. Even better that it's located directly under the range option.

The lib itself doesn't have a distinction in its data based on NPC/Player, so I could not generate that from its data. I used its exact internal setup. I would have to either test the values manually or monitor in-game activity to dynamically update the list based on unit type. The goal was to future proof the list, though.

Also, less colors or no colors? D: